ocp_resources/resource.py

```diff
@@ -279,6 +279,7 @@ class ApiGroup:
         LITMUS_IO = "litmuschaos.io"
         MACHINE_OPENSHIFT_IO = "machine.openshift.io"
         MACHINECONFIGURATION_OPENSHIFT_IO = "machineconfiguration.openshift.io"
+        MAISTRA_ISTIO_IO = "maistra.io"
```

Comment:
Either rename the variable or update the API group string; should match one another, so either:
MAISTRA_ISTIO_IO = "maistra.istio.io"
or
MAISTRA_IO = "maistra.io"

Reply:
Changed to: MAISTRA_IO = "maistra.io"